integration-tests/README.md

```diff
@@ -1,131 +1,224 @@
 # Taquito Integration Tests
 
-The `taquito/integration-tests` directory contains Taquito's integration test suite.
+The `taquito/integration-tests` directory contains the integration test suite for Taquito. These tests are executed against live Tezos testnets, ensuring a comprehensive evaluation of various Taquito use cases.
 
-The integration tests can be run in two environments: a local sandbox, and against a live Tezos testnet.
+## Running Integration Tests Against a Tezos Testnet
 
-## How to Run the Integration Tests Against a Local Sandbox
+To run tests in this environment, make sure you have:
 
-**Note: The recommended method to run tests is against testnets, not sandboxes. Running all tests against a sandbox can fail randomly, while running individual tests usually passes.**
+- Taquito source code cloned to your local machine
+- A compatible version of Node.js installed (see the top-level README.md)
+- Successfully compiled Taquito (see the top-level README)
 
```

Comment:
Here, something is missing. Using the default settings, people outside ECAD cannot run tests. They need to change node URLs.

Reply:
This has been missing, but it's now a good chance to fix it.

Reply:
I will add a third section to the readme to cover running the tets in parallel, how Taquito does it internally, and what external users can do,